src/xSTIR/cSTIR/include/sirf/STIR/stir_data_containers.h

```diff
@@ -513,6 +513,13 @@ namespace sirf {
 		void semibinary_op(const DataContainer& a_x, float y, float(*f)(float, float));
 		void binary_op(const DataContainer& a_x, const DataContainer& a_y, float(*f)(float, float));
 
+		virtual size_t address() const {
+			THROW("data address defined only for data in memory");
+		}
+		virtual size_t address() {
+			THROW("data address defined only for data in memory");
+		}
```

Comment:
can't see why we need a non-`const` version.

Reply:
just removed it